lib/impl/AzureBlobFiles.js

```diff
@@ -177,6 +177,24 @@ class AzureBlobFiles extends Files {
     return azureProps
   }
 
+  async getFileInfo (filePath) {
```

Comment:
I would make this function private and also add it into`lib/Files.js` in the `
  /* **************************** PRIVATE ABSTRACT METHODS TO IMPLEMENT ***************************** */` section. I think it should throw not implemented like other primitives and have some JSDOC to define the  interface (including return object -> typedef)

`getFileInfo` is part of the SPI now, it is a primitive that must be implemented by classes extending Files. (Files and subclasse(s) use the template pattern)